app/views/pages/performance_dashboard/_allocation_content.html.erb

```diff
@@ -1,31 +1,31 @@
-<div class="govuk-grid-row<%= " govuk-!-margin-bottom-3" if recruitment_cycle=="current"%>">
+<div class="govuk-grid-row<%= " govuk-!-margin-bottom-1" if recruitment_cycle=="current"%>">
```

Comment:
```suggestion
<div class="govuk-grid-row<%= recruitment_cycle == "current" ? " govuk-!-margin-bottom-1" : nil %>">
```
I guess it's personal preference, generally if using inline view code ternary is easier to read